Resources/Private/Partials/Frontend/ContentElement/Offers.html

```diff
@@ -0,0 +1,84 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
+<div class="offers">
+    <f:if condition="{offers -> f:count()} > 1">
+        <f:then>
```

Comment:
Ich fänd es übersichtlicher wenn hier im `f:then` und `f:else` jeweils ein `f:render(section` genutzt wird. Dann sieht man auf einen Blick was hier in beiden Fällen passiert, und kann sich dann das Rendering gezielt ansehen.

Aktuell sieht man nur viel Code und weiß nicht ob es nen `f:else` gibt und was dann passiert. Dazu muss man immer den Code überfliegen.